docs/source/changelog.rst

```diff
@@ -42,6 +42,7 @@ Core
 ++++
 
 - Support traversing collections in persist, visualize, and optimize (:pr:`3410`) `Jim Crist`_
+- Add schedule= keyword, replace get= (:pr:`3448`) `Matthew Rocklin`_
```

Comment:
This is a big enough change to deserve more text. In particular "replace" should be explicit: whether `get` is still allowed, or will raise deprecation notice or exception.

Reply:
I've expanded the changelog entry a bit.  It's not yet clear to me what we want to do with the `get=` keyword.  It would be good to get input from more people on this.